user-interface/src/data-verification/ConsolidationOrderAccordion.tsx

```diff
@@ -274,27 +344,42 @@ export function ConsolidationOrderAccordion(props: ConsolidationOrderAccordionPr
   function handleToggleLeadCaseForm(ev: ChangeEvent<HTMLInputElement>): void {
     clearLeadCase();
     setShowLeadCaseForm(ev.target.checked);
+    if (ev.target.checked) {
+      setLeadCaseSelectOptions(getOfficeList(filteredOfficesList ?? props.officesList));
+      if (props.order.childCases) {
+        const selection = getUniqueDivisionCodeOrUndefined(props.order.childCases);
+        if (selection) {
+          setLeadCaseCourt(selection);
+          setLeadCaseSelectValue(selection);
+        }
+      }
+    }
   }
 
   //========== USE EFFECTS ==========
 
   useEffect(() => {
+    let disable: boolean = false;
     if (isConsolidationProcessing) {
-      disableSubmitButtons();
-      clearButtonRef.current?.disableButton(true);
-    } else {
-      enableSubmitButtons();
-      clearButtonRef.current?.disableButton(false);
+      disable = true;
     }
+
+    clearButtonRef.current?.disableButton(disable);
```

Comment:
`function disableButton(targets: ReactRef[]) {
    targets.forEach((target) => {
       target.current?.disableButton(disable);
    }
}`


Then for this handle
`
targetRefs =[clearbuttonRef, rejectButtonRef, approvebuttonRef];
diasbleButton(targetRefs);
`